engine/settings/base.py

```diff
@@ -216,6 +216,7 @@ class DatabaseTypes:
     "apps.public_api",
     "apps.grafana_plugin",
     "apps.webhooks",
+    "apps.metrics_exporter",
```

Comment:
I know this is still in draft, adding this comment just to doublecheck that un-authenticated /metrics/ endpoint won't be exposed outside the cluster before merging